File: 백준/Bronze/2480. 주사위 세개/주사위 세개.py
num1, num2, num3 = input().split()
num1 = int(num1)
num2 = int(num2)
num3 = int(num3)
prize = 0

if num1==num2 and num2==num3:
    prize = 10000 + num1 * 1000
elif num1==num2:
    prize = 1000 + num1 * 100
elif num2==num3:
    prize = 1000 + num2 * 100
elif num1==num3:
    prize = 1000 + num1 * 100
else:
    if num1>num2 and num1>num3:
        prize = num1 * 100
    if num2>num1 and num2>num3:
        prize = num2 * 100
    if num3>num1 and num3>num2:
        prize = num3 * 100

print(prize)

File: 2단계/2480. 주사위 세개/주사위 세개.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args){
        Scanner scanner = new Scanner(System.in);

        int num1 = scanner.nextInt();
        int num2 = scanner.nextInt();
        int num3 = scanner.nextInt();

        int prize = 0;
        if(num1==num2 && num2==num3){
            prize = 10000 + num1 * 1000;
        }else if(num1==num2){
            prize = 1000+num1*100;
        }else if(num2==num3){
            prize = 1000+num2*100;
        }else if(num1==num3){
            prize = 1000+num1*100;
        }else{
            if(num1>num2 && num1>num3){
                prize = num1 * 100;
            }
            if(num2>num1 && num2>num3){
                prize = num2 * 100;
            }
            if(num3>num1 && num3>num2){
                prize = num3 * 100;
            }
        }

        System.out.println(prize);
    }
}
